src/main/scala/org/kframework/minikore/Scanner.scala

```diff
@@ -49,7 +49,7 @@ class Scanner {
           input.next()
         } else if (!yieldEOL) { // end of line
           yieldEOL = true
-          '\n'
+          ' '   // safer than '\n' (platform independent)
```

Comment:
I just tried to eliminate any explicit references to `\n`, because that seems to be platform-specific.  Ideally, I would have liked to replace that with System.lineSeparator(), but that returns a string, because on some platforms the end of line is `\r\n`.